### DISCUSSION TITLE
CefSharp 90 on .NET Framework 4.0

### DISCUSSION THREAD

**Discussion by XHerStar777** [cefsharp/CefSharp#3790] [Q&A]
How to make CefSharp 90 work on .NET Framework 4.0?

**Comment by amaitland** [comment]
Same answer as before see https://github.com/cefsharp/CefSharp/issues/3788#issuecomment-912942545

It is not possible.

**Reply by amaitland** [reply]
The last version to support XP was version 49. This version is unsupported.
